lib/bgp/my_server.ex

```diff
@@ -1,4 +1,4 @@
-defmodule BGP.MyServer do
+defmodule BGP.MyOtherServer do
```

Comment:
Based on the file name it should be:

```suggestion
defmodule BGP.MyServer do
```